### DISCUSSION TITLE
taro build命令在mac系统和win系统打包出来的代码不同，node和taro版本都是一样的

### DISCUSSION THREAD

**Discussion by payne608** [NervJS/taro#11512] [Q & A]
我把在win下打包的dist文件夹和在mac下打包的dist文件夹做了对比，发现win下打包的代码少了几个模块，地图poi搜索页的定位功能用不了，UI样式也有点出入，有没有人可以解答一下
![8K1HCC1QE 0XVA $)X(}SR](https://user-images.githubusercontent.com/74441358/160148519-911142f1-6e9f-4039-9664-991a82c01e11.png)
![8K1HCC1QE 0XVA $)X(}SR](https://user-images.githubusercontent.com/74441358/160148709-4c3128f7-b0bb-4c21-bad7-7266dfe1a6a4.png)

**Comment by Chen-jj** [comment]
提个 issues 附上 demo 咯，只有两个图片 debug 不了。
